docs/lang-ref-ytt.md

```diff
@@ -100,6 +100,66 @@ md5.sum("data") # "8d777f385d3dfec8815d20f7496026dc"
 sha256.sum("data") # "3a6eb0790f39ac87c94f3856b2dd2c5d110e6811602261a9a923d3bb23adc8b7"
 ```
 
+#### Library Loading
+
+ - `load("@ytt:lib", "lib")`
+    ```python
+    lib.get(LIBNAME)
+    ```
+
+    @ytt:lib can be loaded from within starlark and it provides a `lib` module
+    with a `lib.get()` function that returns a library object.
+
+    The library name `LIBNAME` must match one of the forms:
+
+    - `@LIBNAME:[PACKAGE/MODULE]`: loads a library name. The library is found in
+      the `_ytt_lib` directory at the root of the current library.
+
+    - `LIBPATH:[PACKAGE/MODULE]`: loads a library path relative to current
+       package path. The library is loaded relative to the current executing
+       file location.
+
+    - `@:[PACKAGE/MODULE]`: loads the current library. The package specified is
+      loaded relative to the current library root.
+
+    - `PACKAGE/MODULE`: loads a package in the current library relative to the
+       current executing file location.
+
+    If a package and module is specified, only this package and module will be
+    evaluated. Package evaluation is always recursive.
```

Comment:
> If a package and module is specified, only this package and module will be
    evaluated. Package evaluation is always recursive.

we should move this into the next section as this section just talks about format of load paths.